cmd/manager/main.go

```diff
@@ -172,11 +172,9 @@ func serveCRMetrics(cfg *rest.Config) error {
 	if err != nil {
 		return err
 	}
-	// Get the namespace the operator is currently deployed in.
-	operatorNs, err := k8sutil.GetOperatorNamespace()
-	if err != nil {
-		return err
-	}
+	// This is a cluster scope operator so there is no namespace
```

Comment:
The operator has a namespace, its CRs do not. This is used to collect metrics of the operator pod (namespaced under nmstate)

Reply:
Then why this fixed the issue ?

Reply:
Ah, then I misunderstood what is the purpose